features/karaf/src/main/feature/feature.xml

```diff
@@ -244,6 +244,7 @@
   <feature name="esh-storage-mapdb" version="${project.version}">
     <feature>esh-base</feature>
     <bundle>mvn:org.eclipse.smarthome.storage/org.eclipse.smarthome.storage.mapdb/${project.version}</bundle>
+    <bundle dependency="true">mvn:org.mapdb/mapdb/1.0.9</bundle>
```

Comment:
"dependency" means that it is not automatically installed with this feature, is it?
So does it have to be added somewhere in an OH2 base feature as well then?